plugins/GSdx/GSRendererHW.cpp

```diff
@@ -186,6 +186,43 @@ void GSRendererHW::SetGameCRC(uint32 crc, int options)
 	GSRenderer::SetGameCRC(crc, options);
 
 	m_hacks.SetGameCRC(m_game);
+
+	// Code for Automatic Mipmapping. Relies on game CRCs.
+	CRC::Game m_game_mipmap;
+	m_game_mipmap = CRC::Lookup(crc);
```

Comment:
This could be had in a single line, also ``m_`` prefix is only used for member variables, not for local ones.